### ISSUE TITLE
Mask not working as expected

### ISSUE THREAD

**Issue by MathiasHaudgaard** [nemtsov/json-mask#72]
I have the following json
```json
{
  "tom": 3,
  "data1": {
    "first": "Hello",
    "second": {"third": "salude", "fourth": "nvm"}
   },
  "data2": {
    "first": "Bye",
    "second": {"third": "Cheers", "fourth": "nvm"}
   }
}
```
And I'm applying the follwing mask on it.
`"data1(first,second/third),data2(first,second/third)"`

I expected to see the,
```json
{
  "data1": {
    "first": "Hello",
    "second": {"third": "salude"}
   },
  "data2": {
    "first": "Bye",
    "second": {"third": "Cheers"}
   }
}
```
but I'm only getting 
```json
{
  "data1": {
    "first": "Hello",
    "second": {"third": "salude"}
   }
}
```
Can you help me out?

**Comment by nemtsov** [comment]
@MathiasHaudgaard thanks for letting me know! That appears to be a bug. I can reproduce it.

**Comment by MathiasHaudgaard** [comment]
Ok, please let me know when it's fixed. Thanks! :smiley:

**Comment by MathiasHaudgaard** [comment]
@nemtsov It seems like the error comes from the buildTree functions. It exits the scope in the wrong order. When it comes to `")"` it pops `"/"` and now it cant exit the last scope because of 
`if (peek && peek.tag === "/")` doesn't match with `"("`

**Comment by juan-restrepo** [comment]
I'm bumping into the same error right now. Are there any updates here?

**Comment by mihanizm56** [comment]
any updates?

**Comment by alexrainman** [comment]
Is the library abandonware?

**Comment by nemtsov** [comment]
Hey folks, although I don’t have a ton of time to work on this library myself, I’m more than happy to review PRs and help guide people interested in making contributions to it.

**Comment by alexrainman** [comment]
This the only bug i could find so far. Can you fix it?

**Comment by alexrainman** [comment]
What i found is, if / is not at the end of the query, what is next is going to be ignored.

Not working:

```
data1(first,second/third),data2(first,second/third)
```

Working:

```
data1(first,second),data2(first,second/third)
```

**Comment by alexrainman** [comment]
This is the only library that supports partial responses in plan JS. It is a shame just to abandon it.

**Comment by nemtsov** [comment]
This is resolved in **v1.0.4** thanks for reporting it @MathiasHaudgaard
